File: PythonGraphing/Airport.py
from LatLon import LatLon


class Airport(object):
    ''' Instance Variables '''
    code, name, city, state = '', '', '', ''
    centerLatLon = None
    alt = 0.0
    runways = []


    def __init__(self, code, name, city, state, lat, lon, alt):
        self.code = code
        self.name = name
        self.city = city
        self.state = state
        self.centerLatLon = LatLon(lat, lon)
        self.alt = alt
        self.runways = []

    def addRunway(self, runway):
        self.runways.append(runway)
# end class Airport

File: PHP/Airport.php
<?php

require 'LatLon.php';

class Airport
{
        public $code = "";
        public $name = "";
        public $city = "";
        public $state = "";
        public $centerLatLon = NULL;
        public $alt = 0.0;
        public $runways = array();

        function Airport($code, $name, $city, $state, $lat, $lon, $alt)
        {
            $this->code = $code;
            $this->name = $name;
            $this->city = $city;
            $this->state = $state;
            $this->centerLatLon = new LatLon($lat, $lon);
        }

        function addRunway($runway)
        {
            $this->runways.append($runway);
        }
}

?>